### ISSUE TITLE
[Feature Request] Bulk PKG installation Prompt removal

### ISSUE THREAD

**Issue by Crispy81** [shadps4-emu/shadPS4#1580]
Be cool to not have to be prompted for an install directory each time when installing multiple PKG's at once. So installing all PKG's to the same directory, without being prompted, would save some time.

**Comment by ElBread3** [comment]
Good idea! I implemented this feature, so I might just add this once I'm done with my current endeavors.

**Comment by Hermiten** [comment]
@ElBread3 I did assign you :)

**Comment by Hermiten** [comment]
@ElBread3 Any update on this one two ? :)

**Comment by ElBread3** [comment]
> @ElBread3 Any update on this one two ? :)

My upcoming PR for all this is mostly done except for this feature, as I'm still figuring out how exactly to implement it, when I implement that I will PR

**Comment by Hermiten** [comment]
Fixed by https://github.com/shadps4-emu/shadPS4/pull/2261
